src/components/Dashboard/TodoList.vue

```diff
@@ -711,6 +728,10 @@ export default {
 
     loadAlteration (task) {
       const filing: FilingIF = task.task.filing
+      if (!this.isInGoodStanding() && this.isStatusDraft(filing.header)) {
+        task.enabled = false
```

Comment:
This will leave Address Change and Directors Change enabled. Should they be disabled too? If so, search for "has-blocker-task" to see if you reuse that.

Reply:
Well, the task is "Prevent user from starting alteration if "Not In Good Standing"" ...

Reply:
Can you please confirm that COA and COD are allowed?

Reply:
Sienna says "If the client has an alteration draft, and are not in good standing, not only can they not file any other filings because of the draft, but also can't file the alteration draft.
So to answer your question, correct, they cannot file a director or address change. But not because of the "not in good standing" status, it's because they have a draft open."

Reply:
Have you verified that a draft alteration task blocks COA and COD? If it's OK then this conversation can be resolved.

Reply:
verified